src/browser/documentation/help/keys.jsx

```diff
@@ -78,30 +91,24 @@ const content = (
         <tr>
           <td>Change focus to editor</td>
           <td>
-            <div className="key code">/</div>
+            <div className="key code">{printShortcut(FOCUS_SHORTCUT)}</div>
           </td>
         </tr>
         <tr>
           <td>Toggle fullscreen editor</td>
           <td>
-            <div className="key code">Esc</div>
+            <div className="key code">{printShortcut(FULLSCREEN_SHORTCUT)}</div>
           </td>
         </tr>
         <tr>
-          <td />
-        </tr>
-        <tr>
-          <th>Platform specific</th>
-          <th />
-          <th />
-        </tr>
-        <tr>
-          <td>Mac users</td>
```

Comment:
Changed to show the relevant shortcuts directly instead of noting the differences.